components/scream/src/diagnostics/tests/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# NOTE: tests inside this if statement won't be built in a baselines-only build
+if (NOT ${SCREAM_BASELINES_ONLY})
```

Comment:
CMake style: booleans should not be dereferenced with `${..}`. Depending on CMake policy settings, you may get a warning, suggesting you to change this into `if (NOT SCREAM_BASELINES_ONLY)`.

Reply:
okay cool.  I copy/pasted this from elsewhere.  Should I clean up those other CMakeLists.txt files?